### PR TITLE
Use etcd maintainers ML for security related discussion

### PR DESCRIPTION
**Pull request by spzala**
Temporary move to using etcd maintainers mailing list from security@etcd.io.

Related,
https://github.com/kubernetes/community/issues/7739 and https://github.com/kubernetes/k8s.io/pull/6542

A placeholder PR if needed @ahrtr @serathius @wenjiaswe @jmhbnz

### CODE REVIEW

**Review by ahrtr — APPROVED** [review]
lgtm

Thanks @spzala 

Once this PR gets merged, we can notify CNCF staff to terminate the etcd.io Google workspace.

**Comment by spzala** [comment]
> lgtm
> 
> Thanks @spzala
> 
> Once this PR gets merged, we can notify CNCF staff to terminate the etcd.io Google workspace.

Yes, that sounds good, thank you @ahrtr

**Review by jmhbnz — APPROVED** [review]
LGTM - Thanks @spzala

**Comment by ahrtr** [comment]
With the approval of 4 maintainers & sig-etcd leads, I think we can merge this PR. @spzala 

Thanks again.

**Comment by spzala** [comment]
Yes, sounds good @ahrtr Thank you!

**Pull request merged**
